stack-release.html

```diff
@@ -128,7 +148,9 @@ <h1>Weekly Stack Review Meeting Agenda</h1>
 		<li> <a href="https://github.com/Sage-Bionetworks/Synapse-Repository-Services/compare/develop...Sage-Bionetworks:release-#staging#?expand=1" id=plfm_dev_vs_staging style="color:red;">There is content in PLFM staging not merged into develop..</a></li>
 		<li> <a href="https://github.com/Sage-Bionetworks/SynapseWebClient/compare/release-#staging#...Sage-Bionetworks:release-#prod#?expand=1" id=swc_staging_vs_prod style="color:red;">There is content in SWC prod not merged into staging.</a></li>
 		<li> <a href="https://github.com/Sage-Bionetworks/SynapseWebClient/compare/develop...Sage-Bionetworks:release-#staging#?expand=1" id=swc_dev_vs_staging style="color:red;">There is content in SWC staging not merged into develop.</a></li>
-		</ul>
+		<li> <a href="https://github.com/Sage-Bionetworks/Synapse-React-Client/compare/main...Sage-Bionetworks:develop#?expand=1" id=src_dev_vs_main style="color:red;">There is content in synapse-react-client in develop not merged to main.</a></li>
+		<li> <a href="https://github.com/Sage-Bionetworks/SynapseWebClient/blob/develop/package.json" id=src_version_in_swc style="color:goldenrod;">Checking SWC develop for latest version of synapse-react-client...</a></li>
+		</ul> 
```

Comment:
Prefer how you handle src_version_in_swc (initialize in a processing state, then update with either green or red with message).  Rather than the src_dev_vs_main check, which initializes as an error (which is probably how other items on this page work).

Reply:
I'll backfill that behavior for the other checks